sound/soc/sof/Kconfig

```diff
@@ -46,6 +46,15 @@ config SND_SOC_SOF_OF
 	  required to enable i.MX8 devices.
 	  Say Y if you need this option. If unsure select "N".
 
+config SND_SOC_SOF_COMPRESS
+	tristate
+	select SND_SOC_COMPRESS
+	help
+	  This option enables the compress interface that can be used to
+	  offload compress files decoding/encoding on the DSP.
+	  Say Y if you want to enable the compress interface.
+	  If unsure, select "N".
```

Comment:
nit-pick: the help text is not longer relevant and specifically the last two sentences, this is not a user selection.